spec/spacex/launches_spec.rb

```diff
@@ -2,8 +2,11 @@
 
 require 'spec_helper'
 
+RSpec.shared_examples 'Merah Putih' do
```

Comment:
I guess this wasn't suppose to be here. :smiley:

Reply:
Hah, yes that was a left over. Fixing that and the Rubocop issues now.